components/list/list-item-checkbox-mixin.js

```diff
@@ -0,0 +1,76 @@
+import { css, html } from 'lit-element/lit-element.js';
+import { checkboxStyles } from '../inputs/input-checkbox-styles.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { nothing } from 'lit-html';
+
+export const ListItemCheckboxMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			disabled: {type: Boolean },
+			key: { type: String, reflect: true },
+			selectable: {type: Boolean },
+			selected: { type: Boolean, reflect: true }
+		};
+	}
+
+	// TODO: Remove grid styles after list-item-generic is created
```

Comment:
Looks like this comment is no longer needed here?

Reply:
Nope, grid styles aren't meant to live here so this is needed